libs/designsystem/src/lib/kirby.module.ts

```diff
@@ -22,6 +22,7 @@ import { DropdownComponent } from './components/dropdown/dropdown.component';
 import { EmptyStateComponent } from './components/empty-state/empty-state.component';
 import { FabSheetComponent } from './components/fab-sheet/fab-sheet.component';
 import { FlagComponent } from './components/flag/flag.component';
+import { DateInputDirective } from './components/form-field/directives/date/date-input.directive';
```

Comment:
The order is not super well-maintained in this file, but the imports are ordered alphabetically - so this should be under "D"

Reply:
It seemed to be the pre-commit hook which placed it there. Anyways it should be ordered correctly now :)